json-rpc/types/src/views.rs

```diff
@@ -644,18 +644,60 @@ pub struct TransactionView {
     pub vm_status: VMStatusView,
     pub gas_used: u64,
 }
+
 #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
 pub struct TransactionsWithProofsView {
+    // TODO(philiphayes): just serialize as a giant BytesView instead of a Vec<_>?
```

Comment:
I think the question is whether you need create a new type for serializing and deserializing. If you need, it is better to just define the type needed in json-rpc response. Otherwise, maybe we should create BCS-RPC :D

Reply:
going to just leave this as a `Vec<BytesView>` for now